beacon/beacon.go

```diff
@@ -127,6 +127,11 @@ type forwardOrigin func(w http.ResponseWriter, req *http.Request)
 func (f forwardOrigin) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	if origin := req.Header.Get("Origin"); origin != "" {
 		w.Header().Set("Access-Control-Allow-Origin", origin)
+		if req.Method == "OPTIONS" {
+			w.Header().Set("Access-Control-Allow-Methods", "GET,POST")
```

Comment:
should we not handle this separate for each path? "/" will only support GET while "/query" supports both GET and POST

Reply:
Yeah, I was being lazy :) Fixed